CMakeLists.txt

```diff
@@ -194,10 +197,16 @@ set(CMAKE_C_FLAGS_DEBUGOPT "")
 
 target_compile_options(${PROJECT_NAME} PRIVATE -pedantic -std=gnu99 -Wall -Werror -Wimplicit -Wunused -Wcomment -Wchar-subscripts
         -Wuninitialized -Wshadow -Wcast-qual -Wcast-align -Wwrite-strings -Wno-deprecated-declarations -Wno-unknown-pragmas -Wformat-security
-        -Wno-missing-braces -fvisibility=hidden -DS2N_EXPORTS
+        -Wno-missing-braces -DS2N_EXPORTS
```

Comment:
We probably only want to define `S2N_EXPORTS` when paired with `hidden` since [setting it to "default" won't change anything](https://github.com/aws/s2n-tls/blob/main/api/s2n.h#L18)